src/examples.rs

```diff
@@ -0,0 +1,31 @@
+use std::fmt;
+
+use once_cell::sync::Lazy;
+use serde::Deserialize;
+
+#[allow(unused)]
+#[derive(Debug, Deserialize)]
+pub(crate) struct Example {
+    pub(crate) disabled: Option<bool>,
+    pub(crate) slug: String,
+    pub(crate) url: String,
+    #[serde(rename = "desc")]
+    pub(crate) description: String,
+}
+
+impl fmt::Display for Example {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{:#?}", self)
+    }
+}
+
+pub(crate) static EXAMPLES: Lazy<Vec<Example>> = Lazy::new(|| {
```

Comment:
Just define an array of `Example` here directly, we don't gain anything by keeping the examples in toml and parsing at runtime, it only makes this more error-prone

Reply:
Alternatively, even better, add this to the existing benefice config

Reply:
I added a --examples PATH arg so you can override the examples without a deployment. The default Examples.toml is verified with a unit test